src/utils/currencies.ts

```diff
@@ -25,6 +25,8 @@ export const currencyForAnalytics: {
   [CiCoCurrency.cREAL]: 'cReal',
   [CiCoCurrency.ETH]: 'ETH',
 }
+export const symbolToAnalyticsCurrency = (symbol: string): string =>
```

Comment:
Might be easier to follow as a switch statement especially if it grows in the future

```TypeScript
export const symbolToAnalyticsCurrency = (symbol: string): string => {
  switch (symbol) {
    case crealTokenSymbol:
      return 'cReal'
    case celoTokenSymbol:
      return 'cGLD'
    default:
      return symbol
  }
}
```